src/main/resources/view/DarkTheme.css

```diff
@@ -337,12 +337,26 @@
     -fx-background-radius: 0;
 }
 
-#tags {
+#skills {
     -fx-hgap: 7;
     -fx-vgap: 3;
 }
 
-#tags .label {
+#skills .label {
```

Comment:
Ah I see why our styling disappeared. Cause we renamed our tags to requirement and skills. What a CSS god.